kolibri/core/assets/src/views/immersive-app-bar/index.vue

```diff
@@ -0,0 +1,86 @@
+<template>
+
+  <ui-toolbar
+    :title="appBarTitle"
+    textColor="white"
+    type="colored"
```

Comment:
Should this action bar have a different background color?

Reply:
Plan is to make the background color swappable. Actually need that feature soon, so I'm probably going to be adding it. A bit worried, since this uses keen's Sass variables, no?

Reply:
Hmm yes, might have to just use some nested css directly.